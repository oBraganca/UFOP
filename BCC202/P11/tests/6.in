1 Alice 7 12 4 1944
1 Arthur 4 3 7 1965
1 Bernardo 5 13 9 1986
1 Alice 3 2 10 1965
1 Miguel 6 20 9 1940
2
1 Gabriel 6 13 7 1959
1 Pedro 5 17 2 1943
1 Gabriel 1 28 4 1944
1 Davi 9 28 4 1930
1 Pedro 9 8 10 1953
2
1 Pedro 5 7 1 2009
1 Alice 5 7 8 1949
1 Davi 2 6 2 2011
1 Gabriel 9 31 1 2011
1 Helena 1 26 6 1948
1 Isabella 8 9 11 2022
1 Pedro 6 15 5 1972
1 Gabriel 5 17 9 1989
1 Arthur 1 29 7 1984
1 Isabella 8 23 6 1999
1 Sophia 3 1 8 1995
1 Bernardo 2 10 3 1964
1 Sophia 4 18 2 2017
1 Alice 6 28 6 2008
1 Gabriel 7 10 10 1992
2
1 Bernardo 3 5 8 1941
1 Miguel 2 4 3 2011
1 Davi 4 23 2 2013
1 Helena 1 20 1 2017
1 Arthur 1 16 8 1959
2
1 Bernardo 7 30 10 1961
1 Alice 4 4 9 1983
1 Alice 9 9 8 1970
1 Pedro 5 14 4 2006
1 Alice 2 18 6 1967
1 Isabella 1 14 10 1966
1 Sophia 3 16 7 1986
1 Sophia 7 4 2 2023
1 Miguel 6 24 10 1965
1 Gabriel 2 28 9 1983
1 Pedro 6 23 11 1948
1 Helena 3 12 4 1965
1 Miguel 6 23 5 2006
1 Gabriel 6 17 5 1965
1 Pedro 5 16 9 1935
1 Davi 6 5 6 1946
1 Bernardo 3 8 8 1981
1 Bernardo 5 27 9 2019
1 Pedro 9 10 12 1951
1 Miguel 4 14 12 2013
1 Sophia 5 23 3 1943
1 Gabriel 5 10 12 1967
1 Sophia 9 6 5 2002
1 Sophia 5 9 11 1984
1 Bernardo 7 27 6 1996
1 Helena 2 17 6 1972
1 Gabriel 10 26 1 1971
1 Isabella 4 14 8 1939
1 Pedro 6 21 3 1967
1 Gabriel 7 25 3 1980
2
1 Pedro 6 23 1 2002
1 Alice 7 7 6 1944
1 Pedro 6 21 11 1934
1 Arthur 8 29 3 1936
1 Isabella 1 28 2 1996
2
1 Gabriel 6 22 12 2016
1 Arthur 5 27 8 1951
1 Miguel 6 7 10 1961
1 Sophia 9 28 9 1987
1 Helena 4 5 2 1997
1 Bernardo 9 17 4 1987
1 Miguel 9 22 1 1998
1 Miguel 10 5 9 1966
1 Alice 2 8 5 1947
1 Helena 5 27 4 1962
1 Sophia 1 18 1 1951
1 Pedro 9 16 10 1945
1 Gabriel 1 27 2 1950
1 Sophia 9 20 9 1933
1 Pedro 5 13 2 1945
1 Isabella 7 12 10 2015
1 Bernardo 5 3 3 1961
1 Davi 10 6 12 1976
1 Davi 9 28 6 2020
1 Sophia 5 16 4 1952
2
1 Miguel 1 2 2 1949
1 Bernardo 4 1 2 1935
1 Pedro 3 19 8 1941
1 Gabriel 6 26 6 2020
1 Miguel 1 3 12 1972
2
1 Pedro 1 25 3 1982
1 Helena 10 27 2 1973
1 Davi 4 27 8 1967
1 Pedro 2 25 10 1978
1 Alice 1 28 5 1943
1 Bernardo 4 8 8 2021
1 Arthur 8 13 11 1957
1 Bernardo 10 8 8 2010
1 Sophia 10 4 10 1954
1 Miguel 4 21 4 1946
2
3
