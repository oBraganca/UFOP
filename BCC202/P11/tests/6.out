07 | 12/04/1944 | Alice
09 | 28/04/1930 | Davi
09 | 08/10/1953 | Pedro
09 | 31/01/2011 | Gabriel
10 | 26/01/1971 | Gabriel
09 | 10/12/1951 | Pedro
10 | 05/09/1966 | Miguel
10 | 06/12/1976 | Davi
10 | 04/10/1954 | Sophia
10 | 27/02/1973 | Helena
10 | 08/08/2010 | Bernardo
09 | 20/09/1933 | Sophia
09 | 16/10/1945 | Pedro
09 | 09/08/1970 | Alice
09 | 17/04/1987 | Bernardo
09 | 28/09/1987 | Sophia
09 | 22/01/1998 | Miguel
09 | 06/05/2002 | Sophia
09 | 28/06/2020 | Davi
08 | 29/03/1936 | Arthur
08 | 13/11/1957 | Arthur
08 | 23/06/1999 | Isabella
08 | 09/11/2022 | Isabella
07 | 07/06/1944 | Alice
07 | 30/10/1961 | Bernardo
07 | 25/03/1980 | Gabriel
07 | 10/10/1992 | Gabriel
07 | 27/06/1996 | Bernardo
07 | 12/10/2015 | Isabella
07 | 04/02/2023 | Sophia
06 | 21/11/1934 | Pedro
06 | 20/09/1940 | Miguel
06 | 05/06/1946 | Davi
06 | 23/11/1948 | Pedro
06 | 13/07/1959 | Gabriel
06 | 07/10/1961 | Miguel
06 | 17/05/1965 | Gabriel
06 | 24/10/1965 | Miguel
06 | 21/03/1967 | Pedro
06 | 15/05/1972 | Pedro
06 | 23/01/2002 | Pedro
06 | 23/05/2006 | Miguel
06 | 28/06/2008 | Alice
06 | 22/12/2016 | Gabriel
06 | 26/06/2020 | Gabriel
05 | 16/09/1935 | Pedro
05 | 17/02/1943 | Pedro
05 | 23/03/1943 | Sophia
05 | 13/02/1945 | Pedro
05 | 07/08/1949 | Alice
05 | 27/08/1951 | Arthur
05 | 16/04/1952 | Sophia
05 | 03/03/1961 | Bernardo
05 | 27/04/1962 | Helena
05 | 10/12/1967 | Gabriel
05 | 09/11/1984 | Sophia
05 | 13/09/1986 | Bernardo
05 | 17/09/1989 | Gabriel
05 | 14/04/2006 | Pedro
05 | 07/01/2009 | Pedro
05 | 27/09/2019 | Bernardo
04 | 01/02/1935 | Bernardo
04 | 14/08/1939 | Isabella
04 | 21/04/1946 | Miguel
04 | 03/07/1965 | Arthur
04 | 27/08/1967 | Davi
04 | 04/09/1983 | Alice
04 | 05/02/1997 | Helena
04 | 23/02/2013 | Davi
04 | 14/12/2013 | Miguel
04 | 18/02/2017 | Sophia
04 | 08/08/2021 | Bernardo
03 | 05/08/1941 | Bernardo
03 | 19/08/1941 | Pedro
03 | 12/04/1965 | Helena
03 | 02/10/1965 | Alice
03 | 08/08/1981 | Bernardo
03 | 16/07/1986 | Sophia
03 | 01/08/1995 | Sophia
02 | 08/05/1947 | Alice
02 | 10/03/1964 | Bernardo
02 | 18/06/1967 | Alice
02 | 17/06/1972 | Helena
02 | 25/10/1978 | Pedro
02 | 28/09/1983 | Gabriel
02 | 06/02/2011 | Davi
02 | 04/03/2011 | Miguel
01 | 28/05/1943 | Alice
01 | 28/04/1944 | Gabriel
01 | 26/06/1948 | Helena
01 | 02/02/1949 | Miguel
01 | 27/02/1950 | Gabriel
01 | 18/01/1951 | Sophia
01 | 16/08/1959 | Arthur
01 | 14/10/1966 | Isabella
01 | 03/12/1972 | Miguel
01 | 25/03/1982 | Pedro
01 | 29/07/1984 | Arthur
01 | 28/02/1996 | Isabella
01 | 20/01/2017 | Helena
